node-modules-polyfill/src/index.ts

```diff
@@ -52,7 +52,7 @@ export function NodeModulesPolyfillPlugin(
 
                     const resolved = polyfilledBuiltins.get(
```

Comment:
I tried this patch but it didn't work for me because `polyfilledBuiltins` values is the actual polyfill content, not the path, as the rest of this plugin expects.

This changed for this package here: https://github.com/FredKSchott/rollup-plugin-polyfill-node/commit/10ec9b4c9cf3d9323062cce9d7c7f756ab9aba28

I tried the earliest on npm, `0.3.0`, but the above release is already in there.

Reply:
also problematic is that some polyfill contents import `\0polyfill-node._stream_duplex`. Esbuild attempts to find this on disk but the polyfill package doesn't ship this anywhere, so there's no `resolveDir` we could set that would work.

Reply:
@FredKSchott should the imports here be inlined in your collect-polyfills script? https://github.com/FredKSchott/rollup-plugin-polyfill-node/blob/main/polyfills/stream.js#L4 

if not- what's the expected way to configure rollup/esbuild to handle these imports correctly?

Reply:
Sorry for quick & dirty gist delivery, but here is my now-working version of this patch: https://gist.github.com/connorjclark/5535d586af717e1d503cc71de31d840c

Reply:
@connorjclark Can you please add suggestions to this PR to what needs to change in order to make it work as you would expect to?

Reply:
Would it be better to just send a new PR? I'm not sure the original author is still around (and am not convinced it ever worked?)

Reply:
It's pretty much as described in my first two comments. I just checked out this PR as it is locally and ran the tests, and they all failed. Did this patch really work for you?

Reply:
I just updated the package according to the CHANGELOGs

Reply:
@connorjclark I just updated the package according to the CHANGELOGs in @FredKSchott's `rollup-plugin-polyfill-node`

Reply:
@connorjclark I updated the PR to account for the actual content instead of the path